components/Footer.vue

```diff
@@ -0,0 +1,99 @@
+<script setup>
+</script>
+
+<template>
+    <footer class='py-12 bg-main-200'>
+        <div class='flex flex-col justify-center items-center'>
+            <div class='flex flex-row gap-4 items-center mb-6'>
+                <a aria-label='YouTube' target='_blank' href='https://www.youtube.com/@phcos6292'>
+                    <div>
+                        <svg width='48' height='48' viewBox='0 0 48 48' fill='none' xmlns='http://www.w3.org/2000/svg'>
```

Comment:
[Normal]
I think creating components from the SVG images would make `Footer` component more readable.